### PR TITLE
Add more integration tests, fix console errors/warnings

### PR DESCRIPTION
**Pull request by b-pmcg**
### Link to Clubhouse story

https://app.clubhouse.io/dux-makerdao/story/225/end-to-end-testing-of-mvp-delegation

### What does this PR do?

Adds an integration test for the delegations page, covering:
- delegating to a delegate
- undelegating to a delegate
- correct UI updates after actions

Fixes the ugly console warnings and errors
Some updates to other tests

### Steps for testing:

1. Run the tests

### Screenshots (if relevant):

Screen Shot 2021-07-23 at 3 52 38 PM

### CODE REVIEW

File: __tests__/pages/esmodule.js [hunk 1]
```diff
@@ -208,7 +208,9 @@ describe('ES Module', () => {
       expect(el).toBeTruthy();
     });
 
-    test('show disabled Initiate Shutdown button on shutdown initiated', async () => {
+    // TODO: this test passing is a false positive bc 'waitFor' is async, but not awaited.
+    // Also it doesn't appear that the button becomes disabled anyway, that should probably be fixed in esmodule.tsx.
+    xtest('show disabled Initiate Shutdown button on shutdown initiated', async () => {
       const esm = maker.service('smartContract').getContract('MCD_ESM');
```
Comment (rafinskipg) [inline __tests__/pages/esmodule.js:214]: Can we create a ticket in clubhouse for that @b-pmcg ?

File: components/Footer/LongFooter.tsx [hunk 2]
```diff
@@ -71,10 +71,6 @@ export default function LongFooter({ locale = 'en' }: { locale?: string }): Reac
           url: 'https://docs.makerdao.com/dai.js',
           title: 'Dai.js'
         },
-        {
```
Comment (rafinskipg) [inline components/Footer/LongFooter.tsx:74]: Do we want to remove this link from the footer?

**Comment by rafinskipg** [comment]
Nice!

File: components/Footer/LongFooter.tsx (hunk 2 above)
Comment (b-pmcg) [reply components/Footer/LongFooter.tsx:74]: it was a duplicate, the same link to Documentation appears at the top of the list

**Pull request merged**